rotkehlchen/exchanges/binance.py

```diff
@@ -891,8 +900,170 @@ def query_online_trade_history(
 
             trades.append(trade)
 
+        fiat_payments = self._query_online_fiat_payments(start_ts=start_ts, end_ts=end_ts)
+        if fiat_payments:
+            trades += fiat_payments
+            trades.sort(key=lambda x: x.timestamp)
+
         return trades
 
+    def _query_online_fiat_payments(self, start_ts: Timestamp, end_ts: Timestamp) -> List[Trade]:
```

Comment:
Since fiat orders are essentially trades, I simply added them as an additional step when querying binance trades in a specific time range.